pkg/install/stack/usercluster-mla/stack.go

```diff
@@ -0,0 +1,387 @@
+/*
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
unfortunately, we are already 2 years older :)

Reply:
/me - mentally still in 2020 :) thx for noticing

Reply:
done!